src/helpers/script_helpers.sh

```diff
@@ -14,6 +14,7 @@ function command_steps {
   echo_if_all_true "change_directory $(git_root)" "$RUN_IN_GIT_ROOT" "$IN_SUB_FOLDER"
   echo_if_all_true "stash_open_changes" "$STASH_OPEN_CHANGES" "$INITIAL_OPEN_CHANGES"
   steps
+  echo_if_all_true "set_previous_branch \$(expected_previous_branch)"
```

Comment:
They aren't suppose to be "correct". They are just suppose to be all the information you will need to be able to determine what needs to be done.